src/main/java/androidx/util/Base64.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("NOTHING_TO_INLINE")
+
+package androidx.util
+
+import android.util.Base64
+import java.nio.charset.Charset
+
+/**
+ * Convert [String] to base-64 encoded [String].
+ */
+inline fun String.toBase64(
```

Comment:
Let's call this `.encodeBase64` as there's no `Base64` type so we're not really going "to" it.